lib/activegraphql/query.rb

```diff
@@ -0,0 +1,80 @@
+require 'httparty'
+require 'active_support/inflector'
+require 'activegraphql/support/fancy'
+
+module ActiveGraphQL
+  class Query < Support::Fancy
```

Comment:
A little bit late, but how about extracting the formatting out of this class into something like a Mapper that would take a Model and convert it into something that ActiveRecord understands. This mapper would also include methods like name.demodulize.underscore.pluralize.to_sym that is in the Model right now (mapping the name of a Model to the name of the GraphQL model). So we have all the formatting in one place, and it could also be configurable in the future if need be (eg. your GraphQL model name doesn't follow the convention)